src/client/config/local.config.json

```diff
@@ -1,4 +1,5 @@
 {
   "overwriteServerEndpoint": true,
-  "serverEndPointUrl": "localhost"
-}
\ No newline at end of file
+  "serverEndPointUrl": "localhost",
+  "serverPort": 8000
```

Comment:
In local, if broker port is not provided (8000), default port (80) is used (see autobahnConnectionProxy.ts). This change affects local env only. I don't know anything yet about docker side, I'm open to any suggestions. 

As far as I know, FE team is connected to cloud while working locally.